.github/workflows/cibuild.yml

```diff
@@ -40,7 +40,11 @@ jobs:
     - name: Install tox
       run: python -m pip install --upgrade tox
     - name: Python ${{ matrix.python-version }} with astropy
-      run: tox -e py${{ matrix.python-version }}-${{ matrix.astropy-version }}
+      run: |
+        # remove "." from matrix.python-version to get the defaults tox python versions
+        tox_python_target=$(echo py${{matrix.python-version}} | sed 's/\.//')
```

Comment:
I think it is less confusing to set `tox_python_target` as another parameter for the `matrix` above, rather then doing some `sed` calls. This won't work if you want to inject other OS in the future.

Reply:
Not sure I understand what you mean. Add another dimension to the matrix and only run if `python-version` matches the `tox_python_target`?

Reply:
There is some package that enables explicit mapping of GH python version with the `tox` target but it doesn't seem worth it.

Reply:
I was thinking something like https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-including-additional-values-into-combinations but now that I look at it again, I realize maybe that's not what you want.

If this syntax works for you for now, I guess it is okay to deploy as is.

Reply:
It does the trick for now. If it breaks with future version then will address it at that time I guess. Thanks for your quick feedback!